assets/js/base/components/price-slider/index.js

```diff
@@ -308,11 +327,15 @@ const PriceSlider = ( {
 								if ( value === minPriceInput ) {
 									return;
 								}
+								if ( ! Number.isFinite( value ) ) {
+									value = 0;
+								}
```

Comment:
Should this come before the comparison with `minPriceInput` (same question applies for the `maxPriceInput` comparison a bit later)?